packages/terra-props-table/package.json

```diff
@@ -31,7 +31,6 @@
     "classnames": "^2.2.5",
     "commander": "^2.9.0",
     "glob": "^7.1.2",
-    "lodash": "^4.17.4",
```

Comment:
I know we're removing this all together, but for anyone that needs to use lodash in the future, almost every lodash function has it's own package release which is significantly smaller in size.

https://www.npmjs.com/package/lodash.uniq

Reply:
https://www.npmjs.com/search?q=lodash.uniq

Reply:
It will help for sure.